git-hooks/pre-commit

```diff
@@ -7,5 +7,4 @@ do
         exit 1
     fi
 done
-
-$(npm bin)/tsc --noEmit --skipLibCheck
+${BASH_SOURCE%/*}/../node_modules/.bin/ddtsc $(pwd)/tsconfig.json "$(git diff --cached --name-only | grep -E '\.ts$')"
```

Comment:
let's make this an executable of the package so we don't have to do this path dance